sim/mcuboot-sys/build.rs

```diff
@@ -24,6 +24,7 @@ fn main() {
     }
 
     if sig_rsa {
+        conf.define("CONFIG_BOOT_VERIFY_RSA_SIGNATURE", None);
```

Comment:
Let's get rid of this define entire. Instead, change the use of it in the code to MCUBOOT_SIGN_RSA.

Reply:
Done.